app/services/metrc_service/plant/move.rb

```diff
@@ -112,10 +119,16 @@ def next_step(previous_completion = nil, completion = nil) # rubocop:disable Met
       memoize :next_step
 
       def move_plants
-        payload = items.map do |item|
+        barcodes = if is_a_split?
+                     current_completion.options.dig('barcode')
+                   else
+                     items.map { |item| item&.relationships&.dig('barcode', 'data', 'id') }
```

Comment:
If you think we need this else statment we should filter these items by 
```
options.dig('item_ids')
```

Otherwise, it will move all the items in the parent batch.

Reply:
Fixed!